Modules/Core/Common/include/itkNumericTraits.h

```diff
@@ -308,14 +308,14 @@ class NumericTraits<bool> : public std::numeric_limits<bool>
     return !val;
   }
   static constexpr bool
-  IsNegative(bool val)
+  IsNegative([[maybe_unused]] bool val)
   {
-    return val ? false : false;
+    return false;
```

Comment:
Good catch, thanks. Minor detail: in this case I would rather not use `maybe_unused`, as it suggests that the argument is _maybe_ unused. Suggesting that it _might_ be used in some scenario's.  However, in this case the argument is _never_ used anymore. So then I would just remove the argument name.

Reply:
Or use macro `itkNotUsed` instead, e.g. `IsNegative(bool itkNotUsed(val))`

Reply:
If you still want to preserve the argument name, you could use `itkNotUsed`:

    IsNegative(bool itkNotUsed(val))

But again, I would just remove the name `val` in this case.